addons/mail/static/src/models/thread/thread.js

```diff
@@ -811,6 +811,19 @@ function factory(dependencies) {
             return this.pendingFoldState || this.serverFoldState;
         }
 
+        /**
+         * @private
+         */
+        _computeHasSeenIndicators() {
+            if (this.model !== 'mail.channel') {
+                return false;
+            }
+            if (this.mass_mailing) {
+                return false;
+            }
```

Comment:
I have no idea how or why, but the mailing icon is correctly set on the sidebar, but the seen indicator is still showing:

![image](https://user-images.githubusercontent.com/38213504/86924157-5d3b1c00-c12f-11ea-9f9b-805425b8aba0.png)

Reply:
Also it feels like mailing icon is 1 px too low there? Do we actually intend to have it "baseline" visually?

Reply:
baseline puts it too high, has to be centered due to flex on whole row. This will become especially important when we'll have thread icon as avatar (task-2282361)

Reply:
> I have no idea how or why, but the mailing icon is correctly set on the sidebar, but the seen indicator is still showing

Cannot reproduce with change on dependencies, think this was related.